Build fails on powerpc64-unknown-linux-gnu

As of about 3 days ago the build began to consistently fail on powerpc64-unknown-linux-gnu, due to no changes from us.

```
          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /usr/bin/ld: /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: Relocations in generic ELF (EM: 21)

          /home/travis/build/nix-rust/nix/target/powerpc64-unknown-linux-gnu/debug/deps/test_mount-df7f909bdf67289b.13a3d2skisyqlwej.rcgu.o: error adding symbols: File in wrong format

          collect2: error: ld returned 1 exit status
```

https://travis-ci.org/github/nix-rust/nix/jobs/703610987

--- a/.travis.yml
+++ b/.travis.yml
@@ -66,8 +66,6 @@ matrix:
       rust: 1.36.0
     - env: TARGET=powerpc-unknown-linux-gnu DISABLE_TESTS=1
       rust: 1.36.0
-    - env: TARGET=powerpc64-unknown-linux-gnu
-      rust: 1.36.0
     - env: TARGET=powerpc64le-unknown-linux-gnu
       rust: 1.36.0
     - env: TARGET=s390x-unknown-linux-gnu DISABLE_TESTS=1

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -87,6 +87,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Removed `sys::ptrace::ptrace` on Android and Linux.
   (#[1255](https://github.com/nix-rust/nix/pull/1255))
 
+- Dropped support for powerpc64-unknown-linux-gnu
+  (#[1266](https://github.com/nix-rust/nix/pull/1268))
+
 ## [0.17.0] - 3 February 2020
 ### Added
 - Add `CLK_TCK` to `SysconfVar`

--- a/README.md
+++ b/README.md
@@ -58,7 +58,6 @@ Tier 1:
   * mips64-unknown-linux-gnuabi64
   * mips64el-unknown-linux-gnuabi64
   * mipsel-unknown-linux-gnu
-  * powerpc64-unknown-linux-gnu
   * powerpc64le-unknown-linux-gnu
   * x86_64-apple-darwin
   * x86_64-unknown-freebsd

